packages/editor/src/client/panel/publishPagePanel.tsx

```diff
@@ -12,6 +12,7 @@ import {InfoColor} from '../atoms/infoMessage'
 import {DescriptionListItemWithMessage} from '../atoms/descriptionListwithMessage'
 
 export interface PublishPagePanelProps {
+  url?: string | null
```

Comment:
it shouldn't be optional. It's always a string but sometimes it's an empty string.

Reply:
done